src/CoreText/CTFontManager.cs

```diff
@@ -211,8 +227,15 @@ static NSArray EnsureNonNullArray (object [] items, string name)
 			}
 		}
 
-#if !NET
-		[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
+#if NET
+		// [SupportedOSPlatform ("tvos13.0")] - Not valid delegate declaration
```

Comment:
nit - "Not valid on delegate declaration"

Reply:
Nit nipped - https://github.com/xamarin/xamarin-macios/pull/13896/commits/f0bee87d6da7fbbbfb605764d39721d0e3256835